content/community/webinars.md

```diff
@@ -8,12 +8,17 @@ g3Teaser:
   title: Gen3 Webinars
   detail: The Gen3 webinars are focused on helping the community understand what Gen3 does, what the Data Commons Framework Services (DCFS) are, and how both can be used to aid in the construction and operation of data commons.
 g3Upcoming:
+  - Title: Gen3 Data Modeling - Herding Data Submissions & Hunting Down Data (Sheepdog & Peregrine)
+    Date: May 9th, 2019 at 1 PM U.S. Central Time.
+    Description: This webinar will cover the creation of data models, and data dictionaries in Gen3. 
```

Comment:
I don't like the comma but I can get over it

Reply:
This is just something I wrote that we will change later...